gmaps/docs/guides/01-getting-started.md

```diff
@@ -0,0 +1,171 @@
+## Getting started
+
+In this guide, you will learn the basics of visualizing a CARTO dataset with the [Google Maps Javascript API](https://developers.google.com/maps/documentation/javascript/overview). There are no previous requirements to complete this guide, but a basic knowledge of Google Maps Javascript API would be helpful.
+
+After completing this guide, you will have your first Google Maps API map with a CARTO dataset!
+
+<div class="example-map">
+    <iframe
+        id="getting-started-final-result"
+        src="/developers/web-sdk/examples/maps/guides/getting-started/step-2.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+### Basic setup
+
+The first thing you need to do is to add all the required dependencies:
+
+- [Google Maps Javascript API](https://developers.google.com/maps/documentation/javascript/overview) (the API key below is only valid for carto.com so you will need to provide your own API key)
+- [Polyfill.io](https://polyfill.io) 
+- [deck.gl](https://deck.gl), including CARTO [submodule](https://deck.gl/docs/api-reference/carto/overview)
+
+```html
+<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDvHtBZM79O5uGTBT1ZOWOKW2_FVMstHNs&libraries=visualization&v=weekly"></script>
+<script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+<script src="https://unpkg.com/deck.gl@^8.3.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/carto@^8.3.0/dist.min.js"></script>
+```
+
+#### Add map container
+
+Next, you need to create a `div` inside the `body` where the map will be drawn and you need to style them to ensure the map displays at full width:
+
+```html
+<body style="margin: 0; padding: 0;">
+    <div id="map" style="width: 100vw; height: 100vh;"></div>
+</body>
+```
+
+#### Create map and set properties
+
+Once you have a `div` for your map, you can use the `google.maps.Map` constructor to create your map with the desired initial view:
+
+```js
+const map = new google.maps.Map(document.getElementById("map"), {
+    center: {
+        lat: 0,
+        lng: 0,
+    },
+    zoom: 2,
+});
+```
+
+At this point you will have a basic map:
+
+<div class="example-map">
+    <iframe
+        id="getting-started-step-1"
+        src="/developers/gmaps/examples/maps/guides/getting-started/step-1.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+> View this step [here](/developers/gmaps/examples/maps/guides/getting-started/step-1.html)
```

Comment:
Same, url for the full size example is `/developers/gmaps/examples/maps/guides/step-1.html`

```suggestion
> View this step [here](/developers/gmaps/examples/maps/guides/step-1.html)
```

Reply:
I've moved the guide examples to the right folder